packages/destination-actions/src/destinations/insider/insider-helpers.ts

```diff
@@ -32,3 +47,108 @@ export function userProfilePayload(data: Payload) {
     ]
   }
 }
+
+export function sendTrackEvent(data: EventPayload) {
+  const payload: upsertUserPayload = {
+    identifiers: {
+      uuid: data.uuid,
+      custom: {
+        segment_anonymous_id: data.segment_anonymous_id
+      }
+    },
+    attributes: {
+      custom: {}
+    },
+    events: []
+  }
+
+  const events: Object = {
+    'Product List Viewed': 'listing_page_view',
+    'Product Viewed': 'product_detail_page_view',
+    'Product Added': 'item_added_to_cart',
+    'Product Removed': 'item_removed_from_cart',
+    'Email Bounced': 'email_bounce',
+    'Email Delivered': 'email_delivered',
+    'Email Link Clicked': 'email_click',
+    'Email Marked as Spam': 'email_spamreport',
+    'Email Opened': 'email_open',
+    Unsubscribed: 'email_unsubscribe',
+    'Application Opened': 'session_start',
+    'Push Notification Received': 'push_delivered',
+    'Push Notification Tapped': 'push_session',
+    'User Registered': 'sign_up_confirmation'
+  }
+  const defaultAttributes = [
+    'email',
+    'phone',
+    'age',
+    'birthday',
+    'name',
+    'gender',
+    'surname',
+    'city',
+    'country',
+    'app_version',
+    'idfa',
+    'model',
+    'last_ip',
+    'carrier',
+    'os_version',
+    'platform',
+    'timezone',
+    'locale'
+  ]
+  const defaultEvents = [
+    'email_subject',
+    'campaign_id',
+    'campaign_name',
+    'url',
+    'product_id',
+    'user_agent',
+    'ta',
+    'na',
+    'variant_id',
+    'usp',
+    'up',
+    'qu',
+    'piu',
+    'e_quid',
+    'referrer'
+  ]
+
+  const name = events[data.name as keyof Object]
+    ? events[data.name as keyof Object].toString()
+    : data.name.toString().toLowerCase().trim().split(' ').join('_')
+
+  ;(Object.keys(data.attributes || {}) as (keyof typeof data.attributes)[]).forEach((key: string) => {
+    const attributeName: string = key.toString().toLowerCase().trim().split(' ').join('_').toString()
+
+    if (defaultAttributes.indexOf(attributeName) > -1 && data.attributes) {
+      payload.attributes[attributeName as keyof Object] = data.attributes[attributeName]
+    } else if (data.attributes) {
+      payload.attributes.custom[attributeName as keyof Object] = data.attributes[attributeName]
+    }
+  })
+
+  const event: insiderEvent = {
+    event_name: name,
+    timestamp: data.timestamp.toString(),
+    event_params: {
+      custom: {}
+    }
+  }
+
+  ;(Object.keys(data.parameters || {}) as (keyof typeof data.parameters)[]).forEach((key: string) => {
```

Comment:
you can remove `;`

Reply:
when we remove previous lines throw eslint error.

Reply:
I see. Okay then